src/SqlProcessor.php

```diff
@@ -222,8 +222,10 @@ public function processModifier($type, $value)
 
 					case 'i[]':
 						$i = count($value);
-						while ($i-- && is_int($value[$i]));
-						if ($i >= 0) break; // fallback to processArray
+						foreach ($value as $v) {
+							if (is_int($v)) $i--;
+						}
+						if ($i >= 0) break; // fallback to processArray 
```

Comment:
Seems to be a bug :) ans since @JanTvrdik uses if without braces and newline, undetectable by code coverage. Please fix it.